src/Twig/Runtime/RecordRuntime.php

```diff
@@ -134,6 +134,35 @@ public function excerpt($content, $length = 200, $focus = null)
         return $excerpt;
     }
 
+    /**
+     * Find the first image in the given content record.
+     *
+     * This method is deprecated. In Bolt 3.5 it'll be replaced / removed to
+     * fall in line with the work done for #6985
+     *
+     * @param \Bolt\Storage\Entity\Content $content
+     *
+     * @return array image
+     */
+    public function findimage($content)
+    {
+        Deprecated::method('3.4');
```

Comment:
Replaced with?

Reply:
It will be replaced (or integrated with) Ross' work for #6985. It doesn't exist quite yet, so i can't refer to it.